### DISCUSSION TITLE
Install error laravel 9

### DISCUSSION THREAD

**Discussion by salawatbro** [spatie/laravel-pdf#78] [Q&A]
Laravel version: 9.52.16
Php version: 8.2.13
Composer version: 2.6.5

I can't install this package.
The following error occurred while installing the package

![image](https://github.com/spatie/laravel-pdf/assets/70514229/e04288ce-5169-4274-acad-e20b6c361fb0)

**Comment by RVP04** [comment]
laravel-pdf package requires PHP 8.2+, Laravel 10+.
reference: https://spatie.be/docs/laravel-pdf/v1/requirements
